types/marked/tslint.json

```diff
@@ -1,3 +1,7 @@
 {
-    "extends": "@definitelytyped/dtslint/dt.json"
+    "extends": "@definitelytyped/dtslint/dt.json",
+    "rules": {
+        "void-return": false,
+        "no-void-expression": false
```

Comment:
hi @sheetalkamat. Thanks for taking a look at this. I believe I have a valid reason for disabling these rules. The reason is that I wanted to be able to write tests that assert that a return type is `void`. The only way I could figure out how to do that was to write something like 

```typescript
const someVariable: void = functioncall();
```

However, these rules were preventing me from doing that. So, I disabled them not because I wanted to side-step an error in my code, but to allow my test. Do you think that's a valid reason? If not, is there another way you know of to assert that a function overload returns `void`?

Reply:
@acarl005 does it work to add `\\ tslint:disable-next-line:void-return no-void-expression` before each void expression?

Reply:
You can also use `// $ExpectType void` after the call withou assigning it to a varable. https://github.com/Microsoft/dtslint#write-tests